.github/workflows/pr-chart-lint-and-test.yml

```diff
@@ -11,7 +11,7 @@ on:
       - main
 env:
   # see https://github.com/kubernetes-sigs/kind/releases
-  kind_version: v0.18.0
+  kind_version: v0.19.0
```

Comment:
you've to update the image refs in the job matrix as well.
you'll find them in the gh release notes commented alongside.